setup.py

```diff
@@ -11,10 +11,9 @@
     maintainer_email='engineering@close.io',
     description='Efficient datetime mocking in tests',
     test_suite='tests',
-    tests_require=['mock'],
     platforms='any',
     install_requires=[
-        'mock',
+        'mock==2.0.0;python_version<"3"',
```

Comment:
We should not pin `mock` to a specific version here. `mock>=2` could be fine though.

> install_requires is a setuptools setup.py keyword that should be used to specify what a project **minimally** needs to run correctly

> **It is not considered best practice to use install_requires to pin dependencies to specific versions**, or to specify sub-dependencies (i.e. dependencies of your dependencies). This is overly-restrictive, and prevents the user from gaining the benefit of dependency upgrades.

 – https://packaging.python.org/discussions/install-requires-vs-requirements/